tests/integration/long/test_ssl.py

```
@@ -309,33 +324,25 @@ def test_cannot_connect_with_bad_client_auth(self):
         with pytest.raises(NoHostAvailable):
             cluster.connect()
         cluster.shutdown()
 
     def test_cannot_connect_with_invalid_hostname(self):
-        ssl_options = {'ca_certs': CLIENT_CA_CERTS,
-                       'ssl_version': ssl_version,
-                       'keyfile': DRIVER_KEYFILE,
-                       'certfile': DRIVER_CERTFILE}
-        ssl_options.update(verify_certs)
-
         with pytest.raises(Exception):
```

Ruff (0.16.0)

[warning] Do not assert blind exception: `Exception`

(B017)